pkg/rpmmd/repository.go

```diff
@@ -246,23 +246,6 @@ func GetPackage(pkgs []PackageSpec, packageName string) (PackageSpec, error) {
 	return PackageSpec{}, fmt.Errorf("package %q not found in the PackageSpec list", packageName)
 }
 
-func GetVerStrFromPackageSpecList(pkgs []PackageSpec, packageName string) (string, error) {
```

Comment:
Why not leave this one and use it? All the calls to GetPackage are now checking err and doing GetEVRA when there is no error.